content/en/data_observability/jobs_monitoring/_index.md

```diff
@@ -10,6 +12,8 @@ further_reading:
     text: "Ensure trust across the entire data life cycle with Datadog Data Observability"
 ---
 
+<div class="alert alert-info">Data Observability is in Preview.</div>
```

Comment:
Jobs monitoring is not in preview (only Airflow and dbt technologies). Please remove from top level jobs monitoring